08-multi-agent/code_samples/08-python-agent-framework.ipynb

```
@@ -43,7 +49,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "provider = AzureAIProjectAgentProvider(credential=AzureCliCredential())"
+    "provider = create_provider()\n",
+    "print(f\"Provider configured: {describe_provider(provider)}\")"
```

Ruff (0.15.13)

[error] `await` should be used within an async function

(PLE1142)